ci/environment.yml

```diff
@@ -15,6 +15,7 @@ dependencies:
   - universal_pathlib
   - hdf5plugin
   - numcodecs
+  - icechunk>=0.1.1
```

Comment:
Note our primary CI env also now tests against (the last released version of) icechunk, because we can do that now that icechunk and kerchunk are no longer mutually exclusive.

Reply:
It's not on conda just yet

Reply:
Oh right! Will `v0.1.1` be on conda? That would be nice.